packages/base-driver/test/basedriver/driver-e2e-tests.js

```diff
@@ -1,57 +1,60 @@
 import _ from 'lodash';
-import { server, routeConfiguringFunction, DeviceSettings, errors } from '../..';
+import { server, routeConfiguringFunction, DeviceSettings, errors } from '../../index';
 import {
   MJSONWP_ELEMENT_KEY, W3C_ELEMENT_KEY
 } from '../../lib/constants';
 import axios from 'axios';
 import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import B from 'bluebird';
-import getPort from 'get-port';
+import {TEST_HOST, getTestPort} from '../helpers';
 
 const should = chai.should();
-const DEFAULT_ARGS = {
-  address: 'localhost',
-  port: null
-};
 chai.use(chaiAsPromised);
 
 function baseDriverE2ETests (DriverClass, defaultCaps = {}) {
+  const DEFAULT_ARGS = {
+    address: TEST_HOST,
+    port: null
```

Comment:
I would add a short comment about the real meaning of null value here

Reply:
this is outdated